src/features/application/components/Apply.js

```diff
@@ -91,30 +138,51 @@ function Apply() {
     formik.setFieldValue("jobPosition", props.value);
   };
 
+  const handleOnCountryChange = (props) => {
+    formik.setFieldValue("country", props.value);
+    const country = countryInfo.find((country) => country.name === props.label);
+    setCallingCode(country.callingCodes[0]);
+    setFlag(country.flag);
+  };
+
+  React.useEffect(() => {
+    const getCountries = async () => {
```

Comment:
- Generally, this function should be an action, but as we said in this project we will not include raect-redux, leave it for now here.